src/Nethermind/Nethermind.Facade/Simulate/SimulateTxMutatorTracer.cs

```diff
@@ -17,7 +17,7 @@ namespace Nethermind.Facade.Simulate;
 
 internal sealed class SimulateTxMutatorTracer : TxTracer, ITxLogsMutator
 {
-    public const int ExecutionError = -32015;
+    public const int ExecutionError = -32000;
```

Comment:
all error codes are in `Nethermind.JsonRpc.ErrorCodes` class?

Reply:
Done